rke/k8s_version_info.go

```diff
@@ -115,6 +115,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.3.7-rc0",
 			MinRKEVersion:     "1.0.7",
 		},
+		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.18.2-rancher1-1": {
+			MinRancherVersion: "2.3.7-rc0",
```

Comment:
this should be updated, cuz 1.18 is only >=2.4.4

Reply:
Updated in the latest push!